go/cli/mcap/cmd/cat.go

```diff
@@ -68,6 +68,80 @@ type Message struct {
 	Data        json.RawMessage `json:"data"`
 }
 
+func (m Message) Write(w io.Writer) error {
+	_, err := w.Write([]byte("{"))
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write([]byte(`"topic":`))
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write([]byte(`"`))
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write([]byte(m.Topic))
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write([]byte(`",`))
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write([]byte(`"sequence":`))
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write([]byte(strconv.FormatUint(uint64(m.Sequence), 10)))
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write([]byte(`,"log_time":`))
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write([]byte(`"` + strconv.FormatUint(uint64(m.LogTime), 10) + `"`))
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write([]byte(`,"publish_time":`))
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write([]byte(`"` + strconv.FormatUint(uint64(m.PublishTime), 10) + `"`))
```

Comment:
I'm not sure these times are rendered how we want. GH copilot wrote them. Need to double check it.